doc(backup): add full backup support doc (#933)
longhorn/website: content/docs/1.7.0/snapshots-and-backups/scheduling-backups-and-snapshots.md

- The number of backups or snapshots to retain
- The number of jobs to run concurrently
- Any labels that should be applied to the backup or snapshot

> - Parameters that should be applied to the backup or snapshot

Reviewer: Current parameters field only show up when user select `backup` task. Non for snapshot.

Author: fixed. thanks

>   - `full-backup-interval`: Integer parameter that applied only to the backup. With this parameter, Longhorn performs full backup every N incremental backups. For full backup, see [Create a Backup](../backup-and-restore/create-a-backup)

Reviewer 2: This is an important feature for data protection. To effectively promote the feature, we can provide more comprehensive information, including:

- The purpose of the feature.
- Why and when users should enable the feature.
- The benefits of periodic full backups, such as enhanced data protection.
- The drawbacks of periodic full backups, such as increased backup time and cost.

cc @innobead

>   - `full-backup-interval`: Integer parameter that applied only to the backup. With this parameter, Longhorn performs full backup every N incremental backups. For full backup, see [Periodic Full Backup](#periodic-full-backup) and [Create a Backup](../backup-and-restore/create-a-backup).

Author: fixed.

>   - `full-backup-interval`: Number of incremental backups that must be completed before Longhorn performs a full backup. This integer parameter is applied only to the backup. For more information, see [Periodic Full Backup](#periodic-full-backup) and [Create a Backup](../backup-and-restore/create-a-backup).

Reviewer 2: Need to mention the meaning of 0.

Author: fixed.

When the setting is enabled, Longhorn will automatically attach the volume and take a snapshot/backup when it is time to do a recurring snapshot/backup.

Note that during the time the volume was attached automatically, the volume is not ready for the workload. Workload will have to wait until the recurring job finishes.

## Periodic Full Backup

Longhorn performs delta backups by default, which means that only data that was changed since the last backup is uploaded. However, when a data block in the backupstore becomes corrupted, Longhorn does not replace that data block with a healthy one during subsequent backup operations. Corrupted data blocks in the backupstore may cause restoration operations to fail.

> When the `full-backup-interval` parameter is applied, Longhorn performs a full backup every N incremental backups. During a full backup, Longhorn uploads all data blocks in the volume. Data blocks that exist in the backupstore, including corrupted ones, are overwritten.

Reviewer 2: ```suggestion
When a non-zero `full-backup-interval` parameter is set, Longhorn performs a full backup every `full-backup-interval` incremental backups. During a full backup, Longhorn uploads all data blocks in the volume. Data blocks that exist in the backupstore, including corrupted ones, are overwritten.
```

Author: fixed.

> **Important**: 

> > Performing a full backup requires more time and incurs higher network throughput costs compared to the default incremental backup.

Reviewer 2: Add `might` here. If the data in the volume are all overwritten, there is no difference between a full backup and an incremental backup.

```suggestion
> Performing a full backup might take longer and generate higher network throughput and costs than the default incremental backup.
```

Author: fixed.

> By default, Longhorn performs delta backups on snapshots, meaning it backs up only data that has been newly updated since the last backup. However, if a data block in the backup store becomes corrupted, Longhorn does not replace it with a healthy one during subsequent backup operations. This could result in backup restoration failures if a corrupted data block is encountered.
> 
> By applying the `full-backup-interval` parameter, Longhorn performs a full backup every N incremental backups. During a full backup, Longhorn uploads all data blocks of the volume and overrides any existing data blocks in the backup store, thereby automatically fixing corrupted data blocks.
> 
> It's important to note that performing a full backup requires more time and incurs higher network throughput costs compared to the default incremental backup.

Reviewer 3: ```suggestion
Longhorn performs delta backups by default, which means that only data that was changed since the last backup is uploaded. However, when a data block in the backupstore becomes corrupted, Longhorn does not replace that data block with a healthy one during subsequent backup operations. Corrupted data blocks in the backupstore may cause restoration operations to fail.

When the `full-backup-interval` parameter is applied, Longhorn performs a full backup every N incremental backups. During a full backup, Longhorn uploads all data blocks in the volume. Data blocks that exist in the backupstore, including corrupted ones, are overwritten.

> **Important**: 
> Performing a full backup requires more time and incurs higher network throughput costs compared to the default incremental backup.
```

Author: fixed.
